index.html

```diff
@@ -1,11 +1,582 @@
+        </div>
+      </div>
+    </section><!-- End Register Section -->
+
+    <!-- =======  F.A.Q Section ======= -->
+    <section id="faq">
+
+      <div class="container" data-aos="fade-up">
+
+        <div class="section-header">
+          <h2>Frequently Asked Questions</h2>
+        </div>
+
+        <div class="row justify-content-center" data-aos="fade-up" data-aos-delay="100">
+          <div class="col-lg-9">
+            <ul id="faq-list">
+              <!-- <li>
+                <a data-toggle="collapse" href="#faq2" class="collapsed">How do the organizers pick the tasks?<i class="fa fa-minus-circle"></i></a>
+                <div id="faq2" class="collapse" data-parent="#faq-list">
+                  <p>
+                    The tasks were chosen to serve the real time needs of the organization. Thus, your efforts during the competition will directly contribute to the success of PathCheck’s mission to reopen society. 
+                  </p>
+                </div>
+              </li> -->
+
+              <li>
+                <a data-toggle="collapse" href="#faq3" class="collapsed">I will be turning 13 after the beginning of the event, can I still participate? <i class="fa fa-minus-circle"></i></a>
+                <div id="faq3" class="collapse" data-parent="#faq-list">
+                  <p>
+                    Unfortunately, we are only allowing participants who are within the eligible age range on the day of the event. 
+                  </p>
+                </div>
+              </li>
+
+              <li>
+                <a data-toggle="collapse" href="#faq4" class="collapsed">Do I get a certificate? <i class="fa fa-minus-circle"></i></a>
+                <div id="faq4" class="collapse" data-parent="#faq-list">
+                  <p>
+                    MIT Professor Ramesh Raskar will be signing digital certificates for approved essays.
+                  </p>
+                </div>
+              </li>
+
+              <li>
+                <a data-toggle="collapse" href="#faq5" class="collapsed">What if I need help with my task? <i class="fa fa-minus-circle"></i></a>
+                <div id="faq5" class="collapse" data-parent="#faq-list">
+                  <p>
+                    Don't worry! We'll have professional mentors to guide you through your tasks.
+                  </p>
+                </div>
+              </li>
+
+              <li>
+                <a data-toggle="collapse" href="#faq6" class="collapsed">I'm in college, can I participate? <i class="fa fa-minus-circle"></i></a>
+                <div id="faq6" class="collapse" data-parent="#faq-list">
+                  <p>
+                    Unforunately, this event is only being hosted for <b>high schoolers</b>. Stay tuned for more information on future events!
```

Comment:
Let's allow colleg students to attend the presetnations but not submit an essay. you can change the wording based on that.